en/components/combo_cascading.md

```diff
@@ -0,0 +1,164 @@
+</p>
+<div class="divider"></div>
+
+### Demo
+
+The following sample demonstrates scenario, using two igx-drop-down and igx-combo components:
+
+<div class="sample-container loading" style="height: 540px;">
+    <iframe id="cascading-combos-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/cascading-combos" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="cascading-combos-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+### Usage
+To get started with the Ignite UI for Angular Combo import the **IgxComboModule** in the **app.module.ts** file:
+
+```typescript
+// app.module.ts
+
+...
+import { IgxComboModule } from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [..., IgxComboModule],
+    ...
+})
+export class AppModule {}
+```
+
+In the sample below are used an igx-combo and two igx-drop-down components along with corresponding input elements. The API of the both components are used to get selected item from one and load data source for the next either drop down or combo, clear selection or reset data source.
+
+```html
+
+<div class="combo-container">
+    <igx-input-group type="box" #inputGroupCountry style="width: 300px;" (click)="toggleDDCountry()">
+        <input igxInput #inputCountry name="inputCountry" type="text" [(ngModel)]="country" readonly placeholder="Choose Country..."/>
+        <igx-suffix><igx-icon [name]="arrowCountry"></igx-icon></igx-suffix>
+    </igx-input-group>
+    <igx-drop-down #dropdownCountry igxDropDownItemNavigation width="300px"
+        (onSelection)="selectCountry($event)" (onClosed)="onCountryClosed()">
+        <igx-drop-down-item *ngFor="let c of countryData">
+            {{ c }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+
+<div class="combo-container">
+    <igx-input-group type="box" #inputGroupProvince style="width: 300px;" (click)="toggleDDProvince()" [disabled]="disabledProvince">
+        <input igxInput #inputProvince name="inputProvince" type="text" [(ngModel)]="province" readonly placeholder="Choose Province..."/>
+        <igx-suffix><igx-icon [name]="arrowProvince"></igx-icon></igx-suffix>
+    </igx-input-group>
+    <igx-drop-down #dropdownProvince igxDropDownItemNavigation width="300px"
+        (onSelection)="selectProvince($event)" (onClosed)="onProvinceClosed()">
+        <igx-drop-down-item *ngFor="let p of provinceData">
+            {{ p }}
+        </igx-drop-down-item>
+    </igx-drop-down>
+</div>
+
+<div class="combo-container">
+    <igx-combo #comboTown width="300px" [itemsMaxHeight]="225"
+        [data]="townData" [disabled]="true"
+        placeholder="Choose Town..." searchPlaceholder="Search..."
+        [allowCustomValues]="false">
+    </igx-combo>
+</div>
+
+```
+
+```typescript
```

Comment:
We should demonstrate what components and strategies should be imported in the context of the example:
import { IgxComboComponent, IgxDropDownComponent, IgxInputGroupComponent, OverlaySettings, 
 ConnectedPositioningStrategy,  NoOpScrollStrategy  } from 'igniteui-angular';